AudioLibrary.WMME/WaveStream.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.IO;
+using AudioLibrary.WMME.Native;
+
+namespace AudioLibrary.WMME
+{
+    internal class WaveStream : Stream, IDisposable
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Fix this implementation of 'IDisposable' to conform to the dispose pattern. ([csharpsquid:S3881](https://sonarcloud.io/organizations/alex-yevtushenko3820/rules#rule_key=csharpsquid%3AS3881))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=IPP&pullRequest=7&issues=AWfG1-gy8DroMTOoScWy&open=AWfG1-gy8DroMTOoScWy)

[ISSUE_KEY]: AWfG1-gy8DroMTOoScWy